jdbc/build.gradle

```diff
@@ -22,22 +24,44 @@ targetCompatibility = 1.8
 
 repositories {
     mavenCentral()
-    maven { url "https://repo.spring.io/snapshot" }
-    maven { url "https://repo.spring.io/milestone" }
+    maven { url "https://repo.spring.io/release" }
 }
 
 dependencies {
     compile 'org.springframework.cloud:spring-cloud-starter-eureka'
-    compile('org.springframework.cloud:spring-cloud-starter-hystrix:1.0.0.RELEASE')
+    compile('org.springframework.cloud:spring-cloud-starter-hystrix')
     compile("org.springframework:spring-jdbc")
     compile('org.postgresql:postgresql:9.4.+')
     compile('com.zaxxer:HikariCP:2.4.3')
+    compile('commons-lang:commons-lang:2.3')
```

Comment:
I would rather use the newer commons-lang3 version (*org.apache.commons:commons-lang3:3.4*)